src/components/application_manager/include/application_manager/resumption/resume_ctrl_impl.h

```diff
@@ -390,11 +432,30 @@ class ResumeCtrlImpl : public ResumeCtrl,
   void AddWayPointsSubscription(app_mngr::ApplicationSharedPtr application,
                                 const smart_objects::SmartObject& saved_app);
 
+  /**
+   * @brief Checks if saved HMI level is allowed for resumption
+   * by Ignition Cycle restrictions
+   * @param saved_app application specific section from backup file
+   * @return True is allowed , othrwise - False
```

Comment:
@AByzhynar typo
`true if allowed`

Reply:
@LuxoftAKutsan Done. Sorry, commit squashed to old one.